### DISCUSSION TITLE
Nabox Wallet provides integration service proposal for Akash Network

### DISCUSSION THREAD

**Discussion by NaboxAdmin** [akash-network/community#183] [Ideas]
Hey, Akash Network community, This is Nabox multi-chain wallet.The Nabox ecosystem includes but not limited to:Nabox Wallet, Nabox Plug-in, DApp Browser, SwapBox, ParaBox,DID,etc.

We would like to integrate Akash Network, and list Akash Network DApps and assets in Nabox App and plugin wallet. After the integration is completed, we can also do cross-marketing with the Akash Network ecosystem. As one of the trustworthy decentralized wallets, we are backed by a solid community. Inside our wallet, we can list your tokens, can provide hot banners, etc. Please our Partnership Deck: https://drive.google.com/file/d/1o0Vsf2bihufFr20ytZdp8fvCDTP6g2yR/view?usp=sharing

Regarding the Nabox integrated Akash Network service, we plan to apply for $10,000 worth of $AKT as a grant. The grant is primarily used for technology development and long-term maintenance costs, as well as operating and marketing expenses. At the same time, we will give away a packaging service worth $5,000 to the designated Akash Network ecological project.

We hope to get votes and support from the Akash Network community to establish a deep partnership and grow together.

About Nabox：
Nabox is a cross-chain DID application built for Web3.With Nabox, you can manage assets across different blockchains whether for DeFi, GameFi, NFT, and other use cases that come up. Nabox is your gateway to the multi-chain world.

Nabox uses an asymmetric cryptographic algorithm as the basis of Nabox's DID. Users use the same set of keys and public keys to manage assets and data on different blockchains.

Website: nabox.io
Telegram:  t.me/naboxcommunity
Twitter: twitter.com/naboxwallet
Discord: discord.gg/mQVXZJXMkn

**Comment by hewi333** [comment]
Thanks for the putting this in for discussions, just a few questions:

1. Can you go into the value provided to the Akash ecosystem by integrating Nabox. More specifically, how does an integration promote the use or development of open computing resources?
2. How many users/downloads do you have and can you provide documentation of this?
4. $5k is for integration and maintenance. How many dev hours are required? Can some work that has already been made open source by us be used to help?
5. $5k is for marketing. What do your typical marketing campaigns look like for this?

**Reply by NaboxAdmin** [reply]
Thanks for your interest in the proposal. Nabox currently has 300,000+ users, including APK download, GooglePlay, App Store,TestFlight and Chrome plugin wallet. Supporting 4 different languages and serving users from over 20 countries. Nabox will soon support 5 other languages ​​including Japanese, Korean, and Indonesian, and it will take effect when the new version is released.

The standard blockchain integration for Cosmos & EVM Series blockchains is $10,000 worth of token, that covers everything including the one of our package service.

The time for integration development and testing is usually 3-4 weeks.We just need some development documentation on Akash Network. The services include:
1.Nabox and Akash Network will collaborate in blockchain integration and ecosystem construction.
2.Nabox will have Nabox wallet (Mobile version) support Akash Network main-net and all assets based on the Akash network.
3.Nabox will have Nabox wallet (Plug-in) support Akash Network main-net and all assets based on the Akash network.
4.Nabox will have Akash network ecosystem projects listed on Nabox Dapp zone (Mobile version), Nabox wallet users will be able to access Akash network ecosystem DApps in Nabox App.
5.Nabox will have Akash network ecosystem projects listed on Nabox Dapp zone (Plug-in), Nabox wallet users will be able to access Akash ecosystem DApps with Nabox Plug-in.
6.Nabox will give any necessary promotion and community support to Akash Network by displaying relevant information on Nabox wallet banner and Nabox's socia media platforms such as Twitter, Discord, Telegram and Nabox Forum. 
——————————————
Regarding the integration of Akash Network on the Nabox wallet, the $5000 package service provided by the Nabox team includes:
1.Token listing service with logo .
2.Nabox wallet App banner (DApp listing）.
3.Hot DApp display （ 2 weeks）
4.Market/ News/ Pushed ( 5 times)
5.Community events support (Airdrop, AMAs)
6.Partnership Referral,Investor Referral,Additional Product Support, Social Media Support,etc.
7. Providing Nabox wallet banner for Akash Network (2 weeks).

Nabox as one of most trustworthy decentralized multichain wallets around crypto industry. You can check our file deeply and see many projects we have been in good relationship with. https://drive.google.com/file/d/1o0Vsf2bihufFr20ytZdp8fvCDTP6g2yR/view?usp=sharing

We hope to build a long term cooperation to get us a win-win situation.
